app/src/componentsV2/Tabs/components/TabsContainer.tsx

```diff
@@ -49,143 +55,160 @@ export const TabsContainer: React.FC = () => {
       e.preventDefault();
       e.returnValue = "Are you sure?";
     };
-
-    if (hasUnsavedChanges) {
-      window.addEventListener("beforeunload", unloadListener);
-    }
-
+    if (hasUnsavedChanges) window.addEventListener("beforeunload", unloadListener);
```

Comment:
Formatter seems to be creating lots of unnecessary noise which is making the PR hard to review. Can you please remove the unnecessary changes.